neo/Persistence/Memory/Store.cs

```diff
@@ -0,0 +1,70 @@
+using Neo.IO;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+
+namespace Neo.Persistence.Memory
+{
+    public class Store : IStore
+    {
+        private readonly ReaderWriterLockSlim readerWriterLock = new ReaderWriterLockSlim();
+        private readonly Dictionary<byte[], byte[]>[] innerData;
+
+        public Store()
+        {
+            innerData = new Dictionary<byte[], byte[]>[256];
```

Comment:
Set the maxTable instead of 256?

Reply:
It is `byte.MaxValue + 1`.

Reply:
But we don't have 256 tables, we should use less as possible

Reply:
But the table id are not continuous. I don't think there is much difference. Creating an array of 256 elements takes up very little space and time and can be ignored.